andela/taps-client
# #160920081 Instruction of how to get slack ID

#### What does this PR do?
tooltip that shows instructions of how to get slack ID to users
#### Description of Task to be completed?
-Adds tooltip that shows instructions to users
-Adds an input that accepts only alphanumeric
-Wrote a test for ExtraForm component
#### How should this be manually tested?
Navigate to the review app and log in using Andela email. You will see a form with a tooltip.
https://ghoulies-taps-client-pr-10.herokuapp.com/
#### Any background context you want to provide?
This will enable the user to obtain the slack ID from slack
#### What are the relevant pivotal tracker stories?
[#160920081](https://www.pivotaltracker.com/story/show/160920081)
#### Screenshots or gifs (if appropriate)
<img width="1120" alt="screen shot 2018-10-05 at 1 25 53 pm" src="https://user-images.githubusercontent.com/28547970/46535140-5d472180-c8a2-11e8-8a92-dffed857b2d0.png">



#### Questions:
N/A
